frontend/src/components/SearchBar/FindMorePropertiesForm.tsx

```diff
@@ -102,6 +112,13 @@ const FindMorePropertiesForm = <T extends any>(props: any) => {
         </a>
       </p>
       <FormSection>
+        {displayError && (
```

Comment:
I was having some difficulty with out existing components so just used a <p> instead. If preferred I can try to work the other one out

Reply:
the other one requires that an error be in the touched array and in the errors array so that may be all that you are missing.

Reply:
also check that .invalid-feedback is not set do display: none

Reply:
hmm, can't you just trigger handlesubmit and then if there is a validation error, display it?

Reply:
the display being none was the culprit!

Reply:
Do you need the displayError && part? or is it enough that the DisplayError component won't display unless the touched and error values within formik are present?